tests/conftest.py

```diff
@@ -290,6 +290,7 @@ def additional_test_data() -> dict[str, Any]:
     submission = Submission(
         submission_id="TEST-SUBMISSION-ID",
         reporting_round=reporting_round,
+        submission_date=datetime.now(),
```

Comment:
It's generally not a good idea to use test data that can vary on a test-by-test basis, because it leads to complexity like that you've encountered and tried to fix by rounding to the nearest minute (I think).

If we use a static datetime here, does a lot of that complexity and frailty disappear?

Reply:
A random (but set in stone) one - yes :)

Reply:
Changed it to a static datetime.